dist/jquery.floatThead-slim.js

```diff
@@ -937,8 +937,11 @@
           afterPrint();
         }
       };
+
+      var matchMediaPrint;
       if(window.matchMedia && window.matchMedia('print').addListener){
```

Comment:
this still leaks memory right here. I create a new listener in the check for `addListener` (thanks IE)